recipes/libtiff/all/conanfile.py

```diff
@@ -33,15 +32,25 @@ def source(self):
         shutil.copy("CMakeLists.txt",
                     os.path.join(self._source_subfolder, "CMakeLists.txt"))
 
+    def requirements(self):
+        self.requires("zlib/1.2.11")
+        self.requires("zstd/1.4.3")
+        if self.options.lzma:
+            self.requires("xz_utils/5.2.4")
+        if self.options.jpeg:
+            self.requires.add("libjpeg/9c")
```

Comment:
why did I mix? ctrl+x ctrl+v from other recipes ...

Reply:
What do you mean?

Reply:
self.requires and self.requires.add

Reply:
I think it's not that important, both do the same thing